src/sentry/incidents/endpoints/organization_alert_rule_index.py

```diff
@@ -297,7 +297,7 @@ class OrganizationAlertRuleIndexPostSerializer(serializers.Serializer):
     # projects is not required in the serializer, however, the UI requires a project is chosen
     projects = serializers.ListField(
         child=ProjectField(scope="project:read"),
-        help_text="The names of the projects to filter by.",
+        help_text="Metric alerts are currently limited to one project. The array should contain a single slug of the project to filter by.",
```

Comment:
maybe for the last part, instead: "The array should contain a single slug, representing the project to filter by." ?

Reply:
that sounds good to me, updated